actionview/lib/action_view/template/resolver.rb

```diff
@@ -165,7 +169,7 @@ def find_all_with_query(query) # :nodoc:
     # This is what child classes implement. No defaults are needed
     # because Resolver guarantees that the arguments are present and
     # normalized.
-    def find_templates(name, prefix, partial, details, outside_app_allowed = false)
+    def find_templates(name, prefix, partial, details, outside_app_allowed = false, locals = [])
       raise NotImplementedError, "Subclasses must implement a find_templates(name, prefix, partial, details, outside_app_allowed = false) method"
```

Comment:
Does the method signature in this error message need updating?

Reply:
ah yep, I'll fix that